Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,56 @@ label monika_hemispheres_nogets_snow:
     m 3hksdlb "At least you don't have to worry about shoveling it."
     call monika_snow_nogets_snow
     return
-   
+    
+    
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_dream",category=[“life”,”monika”],prompt="I had a dream about you", pool=True))
+
+label monika_dream:
+    m 1sub “Really?!”
+    m wub “That’s amazing [player].”
+    m 1eub “I hope that it was an agreeable dream.”
+    menu:
+        m “Did you manage to come and see me or did I appear in front of you?”
+
+        “You just appear before me.”:
+            m 1hua “Hehehe~…”
+            m “If my love for you achieve to break a barrier between my reality and yours.”
+            m 1eub “Then I’m sure that we will be together someday.”
+            m 1tfu “I just need to try hard enough for that to happen.”
+
+        “I was looking for you.”:
+            m 1wud “…”
+            m 1lkbsa “That’s so thoughtful of you sweetheart.”
+            m 2ekbfb “Your love is so overwhelming that you achieve to create a whole new world just to see me.”
+            m 2ekbfa “You have no idea how much this mean to me [player].” 
+    if not persistent._mas_sensitive_mode:
+        m 1rud “And…”
+        m 1lsbssdrb “I know this is embarrassing to ask, but…”
+        menu:
+            m “Did we…had fun?"
```

Comment:
"have" not had

Reply:
A lot of the typos that you pointed out are from a version that no longer exist.